docs/contributing.rst

```diff
@@ -8,46 +8,114 @@ For a more practical guide to the tedana development, please see our
+what types of data have been collected thus far. These details are subject to
+change, and are intended to provide users with an idea of what is possible,
+rather than definitive recommendations.
+
+Our focus is on functional MRI, including both magnitude and phase data,
+however we understand that quantitative mapping has the potential to aid in
+data processing. Thus, we believe that some details on non-functional MRI
+acquisitions, such as detailed T2* mapping, may fall within the scope of
+tedana. Acquisition related details can be found in the tedana Documentation.
+
+Combining echoes
+----------------
+
+An early step in processing data collected with multiple echoes is the
+combination of the data into a single time series. We currently implement
+multiple options to combine multi-echo data and will add more as they continue
+to be developed. This is an area of active development and interest.
+
+Denoising
+---------
+
+tedana was developed out of a package known as multi-echo ICA (ME-ICA or meica)
+developed by Dr. Prantik Kundu. Though the usage of ICA for classification of
+signal vs noise components has continued in tedana, this is not a rule. The
+tedana community is open and encouraging of new methods, whether or not they
+have a basis in ICA. 
+
+We are interested in any method that seeks to use the information from multiple
+echoes to identify signal (defined here as BOLD signals arising from neural
+processing) and noise (defined here as spurious changes unrelated to neural
+processing, i.e. motion, cardiac, respiration).
+
+tedana is primarily intended to work on volume data, that is, data that is
+still in structured voxel space. Surface-based denoising is not currently
+within the scope of tedana, but code should be written so that it is a
+possible option in the future.
+
+Currently tedana works on a single subject, run by run basis; however, methods
+that use information across multiple runs are welcome. 
+
+Visualization
+-------------
+
+Though tedana does not provide a GUI for inspecting results, it does produce
+figures as part of the processing stream. These figures are intended to help
+users understand the outputs from tedana and diagnose problems. Though a
+comprehensive viewer (such as fsleyes) is outside of the scope of tedana, we
+will continue to improve the figures and add new ones as needed.
+
+Community
+---------
+
+tedana is intended to be a community of multi-echo users. The primary resource
+is the github repository and related documentation. In addition, the tedana
+group will attempt to answer multi-echo related questions on NeuroStars 
+(`multi-echo tag <https://neurostars.org/tag/multi-echo>`_ or
+`tedana tag <https://neurostars.org/tag/tedana>`_). 
+
+What tedana isn’t
+-----------------
+
+While the list of things that do not fall under the scope of tedana are
+infinite, it is worth mentioning a few points:
+
+- tedana will not offer a GUI for usage - it is intended to be either a stand
```

Comment:
I agree with both of these points (no GUI and either standalone or part of eg afni_proc), but I'm not sure how they fit together ?

Reply:
I'm not sure what you're asking about here, sorry, could you clarify? I just figured these were both points of clarification on what we're not adding, rather than related points.

Reply:
Sorry for being unclear ! it seems that they're related since they're under the same bullet point. Were they intended as separate bullet points ? That would make more sense to me

Reply:
Oh! I see. Yes, they're supposed to be separate, my eyes just glossed over that.

Reply:
```suggestion
- tedana will not offer a GUI for usage 
- it is intended to be either a stand
```

Reply:
I think you can go ahead and commit that @handwerkerd